docs/getting-started/tutorial.md

```diff
@@ -0,0 +1,1001 @@
+
+## Adding block attributes
+
+The Copyright Date Block you have built shows the current year, but what if you wanted to display a starting year as well?
+
+![What you're going to build](https://developer.wordpress.org/files/2023/12/block-tutorial-1.png)
+
+This functionality would require users to enter their starting year somewhere on the block. They should also have the ability to toggle it on or off.
+
+You could implement this in different ways, but all of them would require [block attributes](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-attributes/). Attributes allow you to store custom data for the block that can then be used to modify the block's markup.
+
+To enable this starting year functionality, you will need one attribute to store the starting year and another that will be used to tell WordPress whether the starting year should be displayed or not.
+
+### Updating block.json
+
+Block attributes are generally specified in the [`block.json`](https://developer.wordpress.org/block-editor/getting-started/fundamentals/block-json/#data-storage-in-the-block-with-attributes) file. So open up the file and add the following section after the `example` in line 9.
+
+```json
+"attributes": {
+	"showStartingYear": {
+		"type": "boolean"
+	},
+	"startingYear": {
+		"type": "string"
+	}
+},
+```
+
+You must indicate the `type` when defining attributes. In this case, the `showStartingYear` should be true or false, so it's set to `boolean`. The `startingYear` is just a string.
+
+Save the file, and you can now move on to the Editor.
+
+### Updating edit.js
+
+Open the `edit.js` file. You will need to accomplish two tasks.
+
+Add a user interface that allows the user to enter a starting year, toggle the functionality on or off, and store these settings as attributes
+Update the block to display the correct content depending on the defined attributes
+
+#### Adding the user interface
+
+Earlier in this tutorial, you added block supports that automatically created Color and Typography panels in the Settings Sidebar of the block. You can create your own custom panels using the `InspectorControls` component.
+
+##### Inspector controls
+
+The `InspectorControls` belongs to the [`@wordpress/block-editor`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-block-editor/) package, so you can import it into the `edit.js` file by adding the component name on line 14. The result should look like this.
+
+```js
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+```
+
+Next, update the Edit function to return the current block content as well as an `InspectorControls` component that includes the text “Testing.” You must wrap everything in a [Fragment](https://react.dev/reference/react/Fragment) (`<></>`) to ensure proper JSX syntax. The result should look like this.
```

Comment:
A react [Fragment](https://react.dev/reference/react/Fragment) is not really needed, is just an elegant way to wrap up the component without using an extra HTML tag. I would replace "must" by "can"
> You can wrap everything in a [Fragment](https://react.dev/reference/react/Fragment) (`<></>`) to ensure proper JSX syntax.